app/menus/menu.js

```diff
@@ -19,7 +19,11 @@ module.exports = (createWindow, updatePlugins, getLoadedPluginVersions) => {
   const config = getConfig();
   const {commands} = getKeymaps();
 
-  const updateChannel = config.canaryUpdates ? 'canary' : 'stable';
+  let updateChannel = 'stable';
```

Comment:
I think this logic should live inside the config, so that it's populated correctly whereever we need this? But OK for now

Reply:
We have different logics in those two places at the moment, but yep - definitely! 👍